pkg/registry/clusterrolebinding/policybased/storage.go

```diff
@@ -0,0 +1,59 @@
+/*
```

Comment:
> This is also what openshift does (not to say that that makes it appropriate for kubernetes) https://github.com/openshift/origin/blob/388478c40e751c4295dcb9a44dd69e5ac65d0e3b/pkg/authorization/registry/role/policybased/virtual_storage.go

We created our storage long before strategies existed.  Without virtual resources, I don't see a strong case for keeping them.  I think that `Validate` and `ValidateUpdate` provide sufficient power to express the `ConfirmNoEscalation` case.  The cyclical dependency problem due to a lack of separation between a read-only storage and a read-write storage is problematic though.